rc-events-service/src/main/resources/liquibase/changelog/0.0.1/create_event_table.xml

```diff
@@ -10,10 +10,17 @@
         <createTable tableName="event">
             <column name="id" type="${uuid.type}"/>
             <column name="name" type="${64_length_text.type}"/>
+            <column name="description" type="${64_length_text.type}"/>
+            <column name="event_date" type="${timestamp_with_time_zone.type}"/>
+            <column name="creation_date" type="${timestamp_with_time_zone.type}"/>
```

Comment:
[OPT] I would choose to name this field like "createdAt", more readable I think

Reply:
You are right, it will be better